### PR TITLE
Upgrade libraries

### PR DESCRIPTION
**Pull request by javipacheco**
This PR upgrades all libraries and improves the way to include Crashlytics in the project

@fedefernandez can you please review? thanks!

### CODE REVIEW

File: project/Settings.scala [hunk 1]
```diff
@@ -30,7 +28,6 @@ object Settings {
     "-keep class com.google.** { *; }",
     "-keep class com.localytics.android.** { *; }",
     "-keep class com.google.android.gms.ads.** { *; }",
-    "-keep class com.crashlytics.** { *; }",
```
Comment (fedefernandez) [inline project/Settings.scala]: Is this still necessary?

File: project/Settings.scala [hunk 2]
```diff
@@ -15,9 +15,7 @@ object Settings {
       Resolver.sonatypeRepo("snapshots"),
       Resolver.defaultLocal,
       "Scalaz Bintray Repo" at "http://dl.bintray.com/scalaz/releases",
-      "47deg Public" at "http://clinker.47deg.com/nexus/content/groups/public",
-      "zxing Android Minimal" at "https://raw.github.com/embarkmobile/zxing-android-minimal/mvn-repo/maven-repository/",
-      "Crashlytics" at "http://download.crashlytics.com/maven"
```
Comment (fedefernandez) [inline project/Settings.scala]: We need now `"crashlytics" at "https://maven.fabric.io/public"`

**Comment by fedefernandez** [comment]
Besides crashlytics proguard setting and repo, LGTM!

File: project/Settings.scala (hunk 1 above)
Comment (javipacheco) [inline project/Settings.scala]: Yes. It's necessary :+1:

File: project/Settings.scala (hunk 2 above)
Comment (javipacheco) [inline project/Settings.scala]: :+1:

**Pull request merged**